portfolio/src/components/Cover.tsx

```diff
@@ -1,36 +0,0 @@
-import React, { Component } from 'react';
-import profile from "../images/profile.png";
-import Container from 'react-bootstrap/Container';
-
-type Props = {
-
-}
-export default class Cover extends Component<Props>{
-    constructor(props: Props) {
-        super(props);
-    }
-    getProfile() {
-        const backgroundStyles = {
-            backgroundImage: `url(${
-                "https://i.imgur.com/K2zj5sk.png"
```

Comment:
does this have to be an imgur file? You can also import images in react and it will be bundled through webpack under the hood. what is provided by the import is the image location relative as a string. For example:

```tsx
import imageStuff from '../imagelocation/image.png';

function AAA() {
  return <img src={imgStuff} />
}
```